packages/core/src/Provider/StakePoolProvider/types/StakePoolProvider.ts

```diff
@@ -50,6 +50,11 @@ export interface QueryStakePoolsArgs {
    * Will return all stake pools matching the query if not specified
    */
   pagination: PaginationArgs;
+
+  /**
+   * Blockfrost hibryd cache. When enabled, default: true
+   */
+  useBlockfrost?: boolean;
```

Comment:
Can we keep this out of provider interfaces? Shouldn't be a client's concern where it comes from

Reply:
Good catch, I didn't notice this being on the provider interface. It's specific to the `DbSyncStakePoolProvider` implementation, so would be best constrained there.